python/PyQt6/core/auto_generated/providers/arcgis/qgsarcgisportalutils.sip.in

```diff
@@ -25,7 +25,7 @@ Utility functions for working with ArcGIS REST services.
 %End
   public:
 
-    static QVariantMap retrieveUserInfo( const QString &communityUrl, const QString &user, const QString &authcfg, QString &errorTitle /Out/, QString &errorText /Out/, const QgsHttpHeaders &requestHeaders = QgsHttpHeaders(), QgsFeedback *feedback = 0 );
+    static QVariantMap retrieveUserInfo( const QString &communityUrl, const QString &user, const QString &authcfg, QString &errorTitle /Out/, QString &errorText /Out/, const QgsHttpHeaders &requestHeaders = QgsHttpHeaders(), const QString &urlPrefix = QString(), QgsFeedback *feedback = 0 );
```

Comment:
The new arguments will need to be added as the LAST argument to these functions, otherwise we break API for scripts which are specifying the existing feedback argument by position.